server/controllers/userControlers.js

```diff
@@ -0,0 +1,57 @@
+
+const account = require('../models/userCreate')
+
+const userInfo = {
+    userSignUp(req, res) {
+        if ( account == 'undefined') {
+            return res.status(403).json({
+                "status":"error",
+                "error":"undefined mfck"
+            });
+        }
+        else if (account.createUser(req.body) == 'invalid') {
+            console.log(req.body)
+            return res.status(403).json({
+                "status":"error",
+                "error":"you provided invalid information"
+            });
+        }
+        else if (req.body == {}) {
+            return res.status(403).json({
+                "status":"error",
+                "error":"you provided invalid information"
+            }); 
+          
+        } else {
+            return res.status(201).json({
+                "status":"success",
+                "data":{
+                    "token": account.createUser(req.body).token,
+                    "id": account.createUser(req.body).id,
+                    "first_name": account.createUser(req.body).firstName,
+                    "last_name": account.createUser(req.body).lastName,
+                    "email": account.createUser(req.body).email,
+                    "address": account.createUser(req.body).address,
+                    "phoneNumber": account.createUser(req.body).phoneNumber,
+                    "is_admin": account.createUser(req.body).is_admin,
+                    "password": account.createUser(req.body).password
```

Comment:
If you are following proper ES6 syntax. You should make use of object destructuring. As follows:
```javascript
const { id, token, firstName, lastName } = account.createUser(req.body)
...
 return res.status(201).json({
     status: "success",
     "data": {
           token: token,
           id: id,
           firstName: firstName,
           lastName, // if the property and value in an object have the same name, you should use shorthand property assignment.
           ...
});
```
Have a look at this resource if you don't know what it is: